website/docs/extensions/developing/menu.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Menus
+description: Podman Desktop menu reference
+tags: [podman-desktop, extension]
+keywords: [podman desktop, extension, menu]
+---
+
+# Menus
+
+## Menu Details
+
+This section describes how menus are integrated into the extension. These menus are configured to appear in specific parts of the user interface and are tied to commands defined in the extension.
+
+### `package.json` Example
+
+This example shows how to integrate a menu into the Podman Desktop extension through the `package.json` file. Here, a menu item is added under the "dashboard/image" context. Meaning that the command will appear next to the image when `selectImageId` is not in `imagePushInProgressToKind`.
+
+```json
+{
+  "contributes": {
+    "menus": {
+      "dashboard/image": [
+        {
+          "command": "kind.image.move",
+          "title": "Push image to Kind cluster",
+          "when": "selectedImageId not in imagesPushInProgressToKind"
+        }
+      ]
+    }
+  }
+}
+```
+
+### JSON Schema
+
+```json
+{
+  "contributes": {
+    "menus": {
+      "dashboard/<section>": [
```

Comment:
```suggestion
      "<MenuContext>": [
```

May be better to use `MenuContext` ? as we could add more in the future and in the code we are using the wording MenuContext.

https://github.com/containers/podman-desktop/blob/088029ca89fffd040a7ecdc926c3cef06887e874/packages/main/src/plugin/menu-registry.ts#L29-L34